src/App.vue

```diff
@@ -732,23 +732,23 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
         switch (this.getFilingType) {
           case FilingTypes.AMALGAMATION:
             this.$router.push(RouteNames.AMALG_REG_INFORMATION).catch(() => {})
-            break
+            return
```

Comment:
Return so we can retry loading the data on the new route.

Not break from this switch statement and run the remaining code.